verification/types.go

```diff
@@ -1,13 +1,34 @@
 package verification
 
-import "fmt"
+import (
+	"fmt"
+	nsigner "github.com/notaryproject/notation-core-go/signer"
+)
 
 // VerificationType is an enum for signature verification types such as Integrity, Authenticity, etc.
 type VerificationType string
 
 // VerificationAction is an enum for signature verification actions such as Enforced, Logged, Skipped.
 type VerificationAction string
 
+// VerificationResult encapsulates the verification result (passed or failed) for a verification type, including the
+// desired verification action as specified in the trust policy
+type VerificationResult struct {
+	FailedToVerify bool
+	Success        bool
```

Comment:
just out of curiosity, what's the difference between `Success` and `FailedToVerify`

Reply:
Gotcha, thanks for the explanation!